.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,29 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: sudo apt-get install python3-magic
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B301,B303,B311,B403 .
+      - run: black --check . || true
+      - run: codespell --ignore-words-list="tha" --skip="*/text_tests,*/misspelling.json" || true  # Until #51 lands
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=15 --max-line-length=90 --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install ffmpeg SoundFile torchvision -r requirements.txt
+      - run: pip install -e .
+      - run: mypy --ignore-missing-imports . || true
+      # - run: python -m unittest augly.tests.audio_tests.functional_unit_tests
+      - run: python -m unittest discover
+      - run: pytest .
+               --ignore-glob="*/audio_tests/base_unit_test.py"
+               --ignore-glob="*/image_tests/pytorch_test.py"
+               --ignore-glob="*/text_tests/utils_test.py"
+               --ignore-glob="*/video_tests/base_unit_test.py"
+               || true
```

Comment:
im unsure if removing those two imports in the comment above will cause this to fail. if so, i think it's fine to remove this for now